wallet/src/de/schildbach/wallet/ui/send/SendCoinsFragment.java

```diff
@@ -1091,7 +1091,7 @@ else if(Constants.ENABLE_ZERO_FEES && feeCategory == FeeCategory.ECONOMIC || fee
             @Override
             protected void onSuccess(final Transaction transaction) {
                 if (pin != null) {
-                    pinRetryController.successfulAttempt();
+                    pinRetryController.successfulAttempt(config);
```

Comment:
I'm not sure if the user will expect his wallet to be "unlocked" after using the PIN to send a transaction.
I personally would not.

Reply:
I see your point.  No other app does it this way.  Perhaps it could be a setting where the user must choose that behavior.  I haven't heard of any user make comments about this part either way (for or against).  Another story can handle further customization of this feature at a later time.

Reply:
Nevertheless, I thought it would be useful for the user to see their transaction list without unlocking after sending.

Reply:
Didn't think about it. But I agree. I think it's safe to say that users expect to see the tx in the list after sending. 
If @tomasz-ludek agree I think we can let this as it is implemented.